serve_static.ts

```diff
@@ -80,8 +82,12 @@ export function serveStatic(
         "application/octet-stream";
       const headers = new Headers({
         "content-length": stat.size + "",
-        "content-type": contentType,
       });
+      if (opts.withCharSet && opts.withCharSet.includes(contentType)) {
+        headers.set("content-type", `${contentType}; charset=UTF-8`)
```

Comment:
It's not desirable to embed default charset (UTF-8) directly. We need to provide custom configurator function or add charsets automatically based on extension as Express's serve-static does. https://github.com/pillarjs/send/blob/master/index.js#L833